.github/workflows/pr.yml

```diff
@@ -0,0 +1,67 @@
+name: Runs tests
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+  release:
+  workflow_dispatch:
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: vmq_test_user
```

Comment:
In the test, you have used `root`, wondering why they differ.

Reply:
For postgres, vmq_test_user is being used in the tests.